tekton/images/test-runner/runner.sh

```diff
@@ -54,7 +54,7 @@ if [[ "${DOCKER_IN_DOCKER_ENABLED}" == "true" ]]; then
     service docker start
     # the service can be started but the docker socket not ready, wait for ready
     WAIT_N=0
-    MAX_WAIT=5
+    MAX_WAIT=15
```

Comment:
?

Reply:
docker seems to fail to start some times in 5 seconds see: https://storage.googleapis.com/tekton-prow/pr-logs/pull/tektoncd_hub/11/pull-tekton-hub-unit-tests/1278590538689286145/build-log.txt

Reply:
@afrittoli it *is* just to give a little bit more time :stuck_out_tongue_closed_eyes: